integration-cli/requirements.go

```diff
@@ -88,3 +96,14 @@ func testRequires(t *testing.T, requirements ...TestRequirement) {
 		}
 	}
 }
+
+// testRequires checks if the environment satisfies the requirements
+// for the test to run or skips the tests.
+func testRequiresWithLog(t *testing.T, msg string, requirements ...TestRequirement) {
```

Comment:
I'm -1 for this sorry, not only you're duplicating the t.Done() message, you're now missing messages for other `t.Skip`s and other calls to `testRequires`. Either fix them all or fix none, and this really doesn't add any value IMO as it's really obvious in `-test.v`.

Also this messes up `-test.v` output by:

```
[PASSED]: pull - pull fooblahblah1234 (non-existing image)
--- SKIP: TestPullOfficialNames (0.00s)
	requirements.go:87: Actual reason why we fix this test so next line is redundant
[SKIPPED]: pull - pull official names
[PASSED]: pull - pull images from local repo
```